staging/src/k8s.io/apimachinery/pkg/util/json/json_test.go

```diff
@@ -278,42 +280,139 @@ func TestEvaluateTypes(t *testing.T) {
 		},
 	}
 
-	for _, tc := range testCases {
-		inputJSON := fmt.Sprintf(`{"data":%s}`, tc.In)
-		expectedJSON := fmt.Sprintf(`{"data":%s}`, tc.Out)
-		m := map[string]interface{}{}
-		err := Unmarshal([]byte(inputJSON), &m)
-		if tc.Err && err != nil {
-			// Expected error
-			continue
-		}
-		if err != nil {
-			t.Errorf("%s: error decoding: %v", tc.In, err)
-			continue
-		}
-		if tc.Err {
-			t.Errorf("%s: expected error, got none", tc.In)
-			continue
-		}
-		data, ok := m["data"]
-		if !ok {
-			t.Errorf("%s: decoded object missing data key: %#v", tc.In, m)
-			continue
-		}
-		if !reflect.DeepEqual(tc.Data, data) {
-			t.Errorf("%s: expected\n\t%#v (%v), got\n\t%#v (%v)", tc.In, tc.Data, reflect.TypeOf(tc.Data), data, reflect.TypeOf(data))
-			continue
+	for i, tc := range testCases {
+		t.Run(fmt.Sprintf("%d_map", i), func(t *testing.T) {
```

Comment:
this first test-case just got indented into a subtest, viewing without whitespace helps - https://github.com/kubernetes/kubernetes/pull/89833/files?w=1